docs/build.md

```diff
@@ -41,6 +41,17 @@ pylint kglab/*.py
 ```
 
 
+## Security Issues
+
+This project uses [`bandit`](https://bandit.readthedocs.io/) to check
+for *security issues*.
+
+To run these tests:
+```
+bandit kglab/*.py
+```
```

Comment:
really cool to add bandit.

I like to use flake8-bandit to run those tests for example as part of pre-commit hooks https://github.com/louisguitton/simple-cookiecutter-python/blob/master/%7B%7Bcookiecutter.project%7D%7D/.pre-commit-config.yaml#L23

Reply:
Thanks @louisguitton!  Your `simple-cookiecutter-python` is inspirational :)  I have much to dive into for `pre-commit` hooks, and also from the excellent <https://calmcode.io/pre-commit/the-problem.html> by @koaning

Especially where notebooks are getting used as source (though not always the end-use) for integrated documentation, and require more specialized tests.